app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Kindly make your login page look like that of the wireframe provided for this project. 

### Your Log In Page

![Login](https://user-images.githubusercontent.com/77106962/162333965-db86a2eb-2b3c-41f2-abf4-7b99c4fc3434.png)

### Wireframe Sign Up/ Log In Page

![image](https://user-images.githubusercontent.com/77106962/162333548-028aafe3-bcff-436e-bd9f-e094673db53a.png)

#### Note: You can keep the links under the _**login**_ button but style them accordingly. 👍